src/VizzyUtilsContainer.tsx

```diff
@@ -36,27 +34,18 @@ const viz: CustomViz = {
     numDims: Vizzy.makeNumber("Plot", "Number of Dimensions", 1),
     numMeas: Vizzy.makeNumber("Plot", "Number of Measures", 1),
   },
-  // this happens exactly once
```

Comment:
Removes some comments in favor of docstrings... Will continue this in future PRs